build-logic/src/main/kotlin/dokkabuild.java.gradle.kts

```
@@ -26,9 +26,9 @@ tasks.withType<Test>().configureEach {
         (Runtime.getRuntime().availableProcessors() / 2).takeIf { it > 0 } ?: 1
```

```suggestion
        (Runtime.getRuntime().availableProcessors() / 2).coerceAtLeast(1)
```

Some more tidying, since we're here.